### ISSUE TITLE
A question about the difference module

### ISSUE THREAD

**Issue by Herxinsasa** [wgcban/ChangeFormer#38]
HI~，after reading your paper, I still can't understand your design of Difference Module which consists of Conv2D, ReLU and BatchNorm2d,What is the reason for this design? 
in the eqn:
`Fidiff = BN(ReLU(Conv2D3×3(Cat(Fipre, Fipost))))`
in the code：
`        nn.Conv2d(in_channels, out_channels, kernel_size=3, padding=1),
        nn.ReLU(),
        nn.BatchNorm2d(out_channels),
        nn.Conv2d(out_channels, out_channels, kernel_size=3, padding=1),
        nn.ReLU() `
Looking forward to your early reply!

**Comment by wgcban** [comment]
@Herxinsasa 
We adopt a learnable difference module instead of taking the absolute difference between the pre-change and post-change image features. More specifically, we ask the network to automatically learn a proper distance function that might suitable for change detection, which is parameterized by the Conv2D layer. By doing so, we observe a higher performance.

**Comment by Herxinsasa** [comment]
Thank you for your reply：
I understand why using the Cat replace diff ，But  why you design the difference module like this rather than just simple Cat without Conv2D, ReLU and BatchNorm2d.

**Comment by wgcban** [comment]
@Herxinsasa Of course, you can simply use simple concatenation as the difference module; we thought adding a convolution after concatenation would give better results as well as helps to reduce the number of feature maps from 2*C that will slightly improve the speed.

**Comment by Herxinsasa** [comment]
Thank you for your detailed reply！
